testing-services/src/main/java/io/stargate/testing/TestingServicesActivator.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.testing;
+
+import io.stargate.auth.AuthorizationProcessor;
+import io.stargate.core.activator.BaseActivator;
+import io.stargate.testing.auth.AuthorizationProcessorImpl;
+import java.util.Collections;
+import java.util.Hashtable;
+import java.util.List;
+
+public class TestingServicesActivator extends BaseActivator {
+
+  private static final String TEST_AUTHZ_PROCESSOR_ID = "TestAuthzProcessor";
+
+  public TestingServicesActivator() {
+    super("testing-services");
+  }
+
+  @Override
+  protected List<ServiceAndProperties> createServices() {
+    if (TEST_AUTHZ_PROCESSOR_ID.equals(System.getProperty("stargate.authorization.processor.id"))) {
```

Comment:
I am not sure if having the test logic based on the global system property is a good way to test our code.
We are risking that the `createServices` will not create any service and this class client's logic will pass silently without examining the actual service. wdyt?

Reply:
If `stargate.authorization.processor.id` is set, but the implementation is not registered, the persistence will not activate and tests will fail for sure.

Reply:
In general, this seems to be our pattern for swapping service implementations in runtime... we have pretty much the same logic for other "auth" services.